test/Dockerfile

```diff
@@ -10,16 +15,21 @@ ADD skroutz-stable.list /etc/apt/sources.list.d/
 ADD skroutz-pu.list /etc/apt/sources.list.d/
 RUN curl -sSL http://debian.skroutz.gr/skroutz.asc | apt-key add -
 
-RUN apt-get update && \
-    apt-get install -y librdkafka-dev -t stretch-skroutz-proposed-updates && \
+RUN apt-get update &&  \
     apt-get install -y \
-        golang \
-        go-dep \
-        ruby-full \
-        bundler \
-        git \
-        confluent-kafka-2.11 \
-        openjdk-8-jdk
+      golang \
+      go-dep \
+      git \
+      ruby-full \
+      bundler \
+      confluent-kafka-2.11 \
+      default-jre
+
+# build librdkafka
+RUN git clone git://github.com/edenhill/librdkafka /tmp/librdkafka
+WORKDIR /tmp/librdkafka
+RUN git checkout v${RDKAFKA_VERSION}
+RUN ./configure && make && make install
```

Comment:
is there a reason not to use the debian package here?

Reply:
The reason is that we want to be able to test arbitrary versions, including release candidates or the `master` branch.

Reply:
@agis I see your point. Let's skip the `v` prefix then in order to be able to test against branches too. Also, don't forget to add the `v` in the versions in `travis.yml`

Reply:
Oops, nice catch.